CONTRIBUTING.md

```diff
@@ -150,22 +150,30 @@ repository](https://github.com/zulip/zulip/issues?q=is%3Aopen+is%3Aissue+label%3
 
 #### In the main server and web app repository
 
-After making sure the issue is tagged with a [help
+The Zulip server and web app repository
+([`zulip/zulip`](https://github.com/zulip/zulip/)) is set up with a GitHub
+workflow bot called [Zulipbot](https://github.com/zulip/zulipbot), which manages
+issues and pull requests in order to create a better workflow for Zulip
+contributors.
+
+To claim an issue in this repository, simply post a comment that says `@zulipbot claim` to the issue thread. If the issue is tagged with a [help
 wanted](https://github.com/zulip/zulip/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22)
-label, post a comment with `@zulipbot claim` to the issue thread.
-[Zulipbot](https://github.com/zulip/zulipbot) is a GitHub workflow bot; it will
-assign you to the issue and label the issue as "in progress".
+label, Zulipbot will immediately assign the issue to you.
 
-New contributors can only claim one issue until their first pull request is
+Note that new contributors can only claim one issue until their first pull request is
 merged. This is to encourage folks to finish ongoing work before starting
 something new. If you would like to pick up a new issue while waiting for review
 on an almost-ready pull request, you can post a comment to this effect on the
 issue you're interested in.
 
 #### In other Zulip repositories
 
-There is no bot for other repositories, so you can simply post a comment saying
-that you'd like to work on the issue.
+There is no bot for other Zulip repositories
+([`zulip/zulip-mobile`](https://github.com/zulip/zulip-mobile/),
+[`zulip/zulip-terminal`](https://github.com/zulip/zulip-terminal/), etc.). If
```

Comment:
@neiljp is this accurate? I thought terminal used zulipbot, but could be wrong.

Reply:
zulipbot is enabled for terminal, and I thought for most other Zulip repositories?

I'm not sure if we've done a sweep for help-wanted though.